### PR TITLE
Import Solihull (closes #3493)

### LINKED ISSUE #3493: Import SOL-Solihull Metropolitan Borough Council
EMS: Xpress DC
Files:
- `SOL/2021-04-07T11:17:55.688819/Democracy_Club__06May2021.CSV`

### PR DESCRIPTION
**Pull request by alexdutton**

### CODE REVIEW

File: polling_stations/apps/data_importers/management/commands/import_solihull.py [hunk 1]
```diff
@@ -3,91 +3,79 @@
...
+    addresses_name = "2021-04-07T11:17:55.688819/Democracy_Club__06May2021.CSV"
+    stations_name = "2021-04-07T11:17:55.688819/Democracy_Club__06May2021.CSV"
+    elections = ["2021-05-06"]
 
     def station_record_to_dict(self, record):
         # These polling places have a UPRN, and the addressbase postcode doesn't match
         # the postcode from the council. In these cases the addressbase postcode matches
         # the postcode used on the venue's website.
-        # Online references toStation ID 7680 (Whar Hall Road Community Centre) don't
+        # Online references to station ID 7680 (Whar Hall Road Community Centre) don't
```
Comment (GeoWill) [inline polling_stations/apps/data_importers/management/commands/import_solihull.py:15]: Can you drop the station id as it probably doesn't apply anymore.

File: polling_stations/apps/data_importers/management/commands/import_solihull.py [hunk 2]
```diff
@@ -3,91 +3,79 @@
...
+        # Fixes carried forward
+        if record.polling_place_id == "8877":  # Barston Memorial Institute
             record = record._replace(polling_place_postcode="B92 0JU")
-        if record.polling_place_id == "7550":  # St Clements Church
+        if record.polling_place_id == "8907":  # St Clements Church
             record = record._replace(polling_place_postcode="B36 0BA")
-        if record.polling_place_id == "7561":  # Kingshurst Evangelical Church
-            record = record._replace(polling_place_postcode="B37 6NP")
+        # if record.polling_place_id == "7561":  # Kingshurst Evangelical Church
+        #     record = record._replace(polling_place_postcode="B37 6NP")
```
Comment (GeoWill) [inline polling_stations/apps/data_importers/management/commands/import_solihull.py:26]: Can these be uncommented or deleted.

**Review by GeoWill — CHANGES_REQUESTED** [review]
Looks good - couple of changes then good to merge.

File: polling_stations/apps/data_importers/management/commands/import_solihull.py (hunk 1 above)
Comment (alexdutton) [reply polling_stations/apps/data_importers/management/commands/import_solihull.py:15]: Yep

File: polling_stations/apps/data_importers/management/commands/import_solihull.py (hunk 2 above)
Comment (alexdutton) [reply polling_stations/apps/data_importers/management/commands/import_solihull.py:26]: Yep

File: polling_stations/apps/data_importers/management/commands/import_solihull.py (hunk 1 above)
Comment (alexdutton) [reply polling_stations/apps/data_importers/management/commands/import_solihull.py]: I've dropped the ID and reworded for clarity.

File: polling_stations/apps/data_importers/management/commands/import_solihull.py (hunk 2 above)
Comment (alexdutton) [reply polling_stations/apps/data_importers/management/commands/import_solihull.py]: Done

**Pull request merged**